clusters.cnp.example.org/.crossplane/allocatedcluster.ui-schema.yaml

```diff
@@ -0,0 +1,83 @@
+initialConfig: true
```

Comment:
@lukeweber @negz @rathpc @rbwsam what's the alternate proposal and can we get that added to this PR to replace this?

Reply:
Is it a goal to arrive at our ideal "north star" solution for this before we merge this PR? I'd be happy to remove this for now - I think we already have an issue in this repo to figure out how we solve the broader problem of creating things when installing a package.

Reply:
@lukeweber @negz would be great to frame up the alternate proposal either in this PR, or in a follow-on PR and/or UI meta branch. /cc @rathpc @rbwsam

Reply:
I'm going to remove initialConfig for now since we seem unsure about it. I think https://github.com/crossplane/example-cnp/issues/4 could be a good place to hash this out.